apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/FissClaimTransformerTest.java

```diff
@@ -0,0 +1,201 @@
+package gov.cms.bfd.pipeline.rda.grpc.source;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.samePropertyValuesAs;
+import static org.junit.Assert.*;
+
+import gov.cms.bfd.model.rda.PreAdjFissClaim;
+import gov.cms.bfd.model.rda.PreAdjFissProcCode;
+import gov.cms.bfd.pipeline.sharedutils.IdHasher;
+import gov.cms.mpsm.rda.v1.FissClaim;
+import gov.cms.mpsm.rda.v1.FissProcCodes;
+import java.math.BigDecimal;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FissClaimTransformerTest {
+  // using a fixed Clock ensures our timestamp is predictable
+  private final Clock clock = Clock.fixed(Instant.ofEpochMilli(1621609413832L), ZoneOffset.UTC);
+  private final FissClaimTransformer transformer =
+      new FissClaimTransformer(clock, IdHasher.createInstanceForTestingOnly());
```

Comment:
Should we mock the hashing utility object instance so we're not testing code from a different class that has it's own tests?

Reply:
The type of hashing used is directly relevant to this class so I like to include it.  If someone changed the hasher it would break this class.  Also I like to use the minimal number of mocks to get coverage.